src/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "Elao : Vagrant Extension",
-  "version": "0.0.2",
-  "manifest_version": 2,
+  "version": "0.0.3",
+  "manifest_version": 3,
```

Comment:
Nope, manifest_version is the # of [manifest specification](https://developer.chrome.com/extensions/manifestVersion) you choose to use, version 2 in that case